### DISCUSSION TITLE
[Feature Request][Gits] Download file from gits friendly url

### DISCUSSION THREAD

**Discussion by spaccabit** [community/community#24212] [Repositories]
<p>Hi,</p>
<p>I would suggest adding the friendly URL for downloading a file from gits.</p>
<p>Example:</p>
<p>the url https://gist.githubusercontent.com/spaccabit/2e78b0a19b698e715d85e0a644f86e89/raw/2af2106c2b4dedc023eba4ad25ba59bb2b522b42/Builds.json</p>
<p>could turn into: <a href="https://rows.gist.github.com/spaccabit" target="_blank" rel="nofollow noopener">https://rows.gist.github.com/spaccabit</a>/[group file name]/Builds.json</p>
<p>that point to the latest version of the file.</p>

**Comment by nyahbhinghiPrincess** [comment, marked as answer]
<p>Hi <a class="mention" href="/spaccabit">@spaccabit</a>,</p>
<p>Thank you for that great eye and for the feedback! I’ve taken your suggestion and passed it along to the appropriate teams.<br>
I don’t have an ETA on when or if this request will be implemented but this is in good hands.</p>
